specs/commandline/plugin.md

```diff
@@ -41,51 +42,87 @@ Aliases:
 ### notation plugin install
 
 ```text
-Installs a plugin
+Install a plugin
 
 Usage:
-  notation plugin install [flags] <plugin path>
+  notation plugin install [flags] <plugin location>
 
 Flags:
   -h, --help                    help for install
   -f, --force                   force the installation of a plugin
-
+      --checksum                verify the checksum digest of a plugin
+      --source string           the location of plugin installation file, options: "file", "url","registry" (default "file")                  
 Aliases:
   install, add
 ```
 
-### notation plugin remove
+### notation plugin uninstall
 
 ```text
-Removes a plugin
+Uninstall a plugin
 
 Usage:
-  notation plugin remove [flags] <plugin name>
+  notation plugin uninstall [flags] <plugin name>
 
 Flags:
   -h, --help          help for remove
 
 Aliases:
-  remove, rm, uninstall, delete
+  remove, rm, uninstall
 ```
 
 ## Usage
 
-### Install a plugin
+## Install a plugin 
+
+### Install a plugin from a local file
+
+```shell
+$ notation plugin install <path> --checksum <digest>
+```
+
+Upon successful execution, the plugin is copied to plugin directory. The name and version of the installed plugin is displayed as follows. 
+
+```console
+Successfully installed plugin <plugin name>, version <x.y.z>
+```
+
+If the plugin directory does not exist, it will be created. When an existing plugin is detected and the version is the same as the installing plugin, it fails to install and returns the error as follows. Users can use a flag `--force` to skip version check and force the installation a specified version.
```

Comment:
Since we have the `notation plugin upgrade` command, we are not checking versions in the `notation plugin install` command anymore. Instead, if there is an existing plugin, the install would fail.
```suggestion
If the plugin directory does not exist, it will be created. When an existing plugin is detected, it fails to install and returns the error as follows. Users can use a flag `--force` to skip existence check and force the installation with a specified version.
```